sda/internal/database/database.go

```diff
@@ -54,6 +54,12 @@ type SubmissionFileInfo struct {
 	CreateAt  string `json:"createAt"`
 }
 
+type DatasetInfo struct {
+	DatasetID string
+	Status    string
+	Timestamp string
```

Comment:
```suggestion
	DatasetID string `json:"datasetID"`
	Status    string `json:"status"`
	Timestamp string `json:"timeStamp"`
```

Reply:
When file info are listed, the keys are all started with lower case letter, for consistency, it is better to keep the same format for `DatasetInfo` as well. 

In addition, for the variable `Timestamp`, will it be modified after creation? Maybe `CreateAt` is a better name?

Reply:
It's not the creation date that are of interest, it's the timestamp for the last status change.